### PR TITLE
IBX-6702: BasePriceStatsAggregation and CustomPriceStatsAggregation described

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-6702)
| Versions      | master
| Edition       | Content/Headless, Experience, Commerce

BasePriceStatsAggregation and CustomPriceStatsAggregation described in Developer Documentation.

Related PR: https://github.com/ibexa/product-catalog/pull/1075

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Code samples are working
- [x] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/search/aggregation_reference/basepricestats_aggregation.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- sum (`getSum()`)
+- count of values (`getCount()`)
+- minimum value (`getMin()`)
+- maximum value (`getMax()`)
+- average (`getAvg()`)
+
+## Arguments
+
+- `name` - name of the Aggregation
+- `currencyCode` - currency code of the price
```
Comment (adamwojs) [inline docs/search/aggregation_reference/basepricestats_aggregation.md:15]: `BasePriceStatsAggregation` expects `\Ibexa\Contracts\ProductCatalog\Values\CurrencyInterface` instance as second. arg

File: docs/search/aggregation_reference/custompricestats_aggregation.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- sum (`getSum()`)
+- count of values (`getCount()`)
+- minimum value (`getMin()`)
+- maximum value (`getMax()`)
+- average (`getAvg()`)
+
+## Arguments
+
+- `name` - name of the Aggregation
+- `currencyCode` - currency code of the price
```
Comment (adamwojs) [inline docs/search/aggregation_reference/custompricestats_aggregation.md:15]: `Ibexa\Contracts\ProductCatalog\Values\Product\Query\Aggregation\CustomPriceStatsAggregation` expects \Ibexa\Contracts\ProductCatalog\Values\CurrencyInterface instance as second. arg

File: docs/search/aggregation_reference/custompricestats_aggregation.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- `customerGroup` - customer group that defines custom pricing
```
Comment (adamwojs) [inline docs/search/aggregation_reference/custompricestats_aggregation.md:16]: Please add information about default value (customer group assigned to current user)

**Review by adamwojs — CHANGES_REQUESTED** [review]

File: docs/search/aggregation_reference/custompricestats_aggregation.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- count of values (`getCount()`)
+- minimum value (`getMin()`)
+- maximum value (`getMax()`)
+- average (`getAvg()`)
+
+## Arguments
+
+- `name` - name of the Aggregation
+- `\Ibexa\Contracts\ProductCatalog\Values\CurrencyInterface` - currency of the price
+- `customerGroup` - customer group that defines custom pricing, by default it is the one assigned to current user
```
Comment (adamwojs) [inline docs/search/aggregation_reference/custompricestats_aggregation.md:16]:
```suggestion
- `\Ibexa\Contracts\ProductCatalog\Values\CustomerGroupInterface|null` - customer group that defines custom pricing, by default it is the one assigned to current user
```

**Review by adamwojs — APPROVED** [review]

File: docs/search/aggregation_reference/custompricestats_aggregation.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+- count of values (`getCount()`)
+- minimum value (`getMin()`)
+- maximum value (`getMax()`)
+- average (`getAvg()`)
+
+## Arguments
+
+- `name` - name of the Aggregation
+- `\Ibexa\Contracts\ProductCatalog\Values\CurrencyInterface` - currency of the price
+- `\Ibexa\Contracts\ProductCatalog\Values\CustomerGroupInterface|null` - customer group that defines custom pricing, by default it is the one assigned to current user
```
Comment (adriendupuis) [inline docs/search/aggregation_reference/custompricestats_aggregation.md:16]:
```suggestion
- `name` (`string`) - name of the Aggregation
- `currency` (`\Ibexa\Contracts\ProductCatalog\Values\CurrencyInterface`) - currency of the price
- `customerGroup` (`\Ibexa\Contracts\ProductCatalog\Values\CustomerGroupInterface` or `null`) - customer group that defines custom pricing, by default it is the one assigned to current user
```

File: docs/search/aggregation_reference/basepricestats_aggregation.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+- sum (`getSum()`)
+- count of values (`getCount()`)
+- minimum value (`getMin()`)
+- maximum value (`getMax()`)
+- average (`getAvg()`)
+
+## Arguments
+
+- `name` - name of the Aggregation
+- `\Ibexa\Contracts\ProductCatalog\Values\CurrencyInterface` - currency of the price
```
Comment (adriendupuis) [inline docs/search/aggregation_reference/basepricestats_aggregation.md:15]:
```suggestion
- `name` (`string`) - name of the Aggregation
- `currency` (`\Ibexa\Contracts\ProductCatalog\Values\CurrencyInterface`) - currency of the price
```

File: docs/search/aggregation_reference/custompricestats_aggregation.md (hunk 5 above)
Comment (adriendupuis) [reply docs/search/aggregation_reference/custompricestats_aggregation.md:16]: @adamwojs What do you think of those changes to not mix arguments names and arguments types?

**Review by juskora — APPROVED** [review]

**Pull request merged**